rkt/run.go

```diff
@@ -62,15 +62,40 @@ End the image arguments with a lone "---" to resume argument parsing.`,
 	flagLocal       bool
 )
 
+//
+// Search a series of directories for a stage1 image
+//
+
+var stage1ImagePath = []string{
+	"./",
```

Comment:
The first element was really a placeholder so that I could set the results of the proc lookup without resizing or regenerating the slice.  I had it there so I could maintain the current behavior of looking in the dirname for the rkt binary.

I could have it fail entirely if the proc path discovery fails.